console/console-init/ui/src/modules/address-space/components/AddressSpaceConfiguration/AddressSpaceConfiguration.tsx

```diff
@@ -3,7 +3,7 @@
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
 
-import * as React from "react";
+import React from "react";
```

Comment:
Any specific reason to import React instead * as React statement

Reply:
Difference between *as and normal import is * as import includes all the modules and normal import includes default imports only. At this point we need only default imports not whole modules.